platform/pensando/dsc-drivers/src/drivers/linux/pciesvc/README.md

```diff
@@ -0,0 +1,22 @@
+# pciesvc module
+
+## Overview
+
+This driver module is a companion to the kpcimgr driver.  This module
```

Comment:
where is this driver being packaged into the image? i do not see it.

Reply:
We are not loading this currently. Will be adding functionality to load this later.